packages/core/package.json

```diff
@@ -30,13 +30,14 @@
     "d3-time-format": "^2.1.3",
     "lodash": "^4.17.11",
     "react-spring": "9.1.2",
-    "recompose": "^0.30.0",
     "resize-observer-polyfill": "^1.5.1"
   },
   "devDependencies": {
+    "@nivo/recompose": "0.68.0",
```

Comment:
This should just be in the dependencies. We have tooltip as dev + peer because currently, `core` and `tooltip` depend on each other.

Reply:
cheers, I was wondering about that - will change.